docs/pages/flexbox.md

```diff
@@ -6,7 +6,7 @@ sass:
   - scss/util/_flex.scss
 ---
 
-Foundation components use a combination of floats, vertical alignment, table cells, and various other CSS hacks to get layouts looking right. These days, there's a better way... if you have the browser support!
+Foundation components use a combination of floats, vertical alignment, table cells, and various other CSS hacks to get layouts looking right. These days though, there's a better way... if you are happy with the below browser support!!
```

Comment:
Ok, one more piece of git/github practice for you. :)  You added an extra '!' at the end... lets remove it. You should be able to update this PR simply by pushing another commit on the same branch.